services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,39 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
```

Comment:
check to make sure queue has already been created

Reply:
The queue is being created in initThermalSystemManager(). Should I be editing this function? And if I wanted to make sure the queue has been created, how would I do so?

Reply:
you can just check in this function if `thermalMgrQueueHandle == NULL` since if it's not created it will be initialized to NULL since it's a static variable